OpenRA.Mods.Common/Traits/EngineerRepair.cs

```diff
@@ -29,21 +33,32 @@ class EngineerRepairInfo : ITraitInfo
 		[Desc("What diplomatic stances allow target to be repaired by this actor.")]
 		public readonly Stance ValidStances = Stance.Ally;
 
-		public object Create(ActorInitializer init) { return new EngineerRepair(init, this); }
+		[Desc("Sound to play when repairing is done.")]
+		public readonly string RepairSound = null;
+
+		[Desc("Cursor to show when hovering over a valid actor to repair.")]
+		public readonly string Cursor = "goldwrench";
+
+		[Desc("Cursor to show when target actor has full health so it can't be repaired.")]
+		public readonly string FullHealthCursor = "goldwrench-blocked";
```

Comment:
Please call this `RepairBlockedCursor` for consistency with our other trait cursor fields.